src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -321,51 +333,40 @@ public void testConfigureClientHappyCase() throws Exception {
     assertEquals(0L, configureClientResponse.getClientSequencer().longValue());
   }
 
+  @Ignore
   @Test
   public void testConfigureClientNoPipeFound() throws Exception {
     final String userAgentSuffix = "kafka-provider/NONE";
     SimpleIngestManager manager = TestUtils.getManager("nopipe", userAgentSuffix);
     try {
       manager.configureClient(null);
     } catch (IngestResponseException exception) {
-      assertEquals(
-          exception.getMessage(),
-          "HTTP Status: 404 ErrorBody: {\n"
-              + "  \"data\" : null,\n"
-              + "  \"code\" : \"390404\",\n"
-              + "  \"message\" : \"Specified object does not exist or not authorized. Pipe not found\",\n"
-              + "  \"success\" : false,\n"
-              + "  \"headers\" : null\n"
-              + "}");
+      assertEquals(404, exception.getErrorCode());
+      assertEquals("Specified object does not exist or not authorized. Pipe not found",  exception.getErrorBody().getMessage());
     }
   }
 
+  @Ignore
   @Test
   public void testGetClientStatusHappyCase() throws Exception {
     final String userAgentSuffix = "kafka-provider/NONE";
     SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix);
     manager.configureClient(null);
     ClientStatusResponse clientStatusResponse = manager.getClientStatus(null);
     assertEquals(0L, clientStatusResponse.getClientSequencer().longValue());
-    assertEquals(null, clientStatusResponse.getOffsetToken());
+    assertNull(clientStatusResponse.getOffsetToken());
```

Comment:
umm, wont this be `null` string? or the `ClientStatusResponse` deserializes it into null?

Reply:
yah, assertNull makes sure it is null